src/MonthView/phone/ScrollListTaskComponent/ScrollListTaskComponent.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet, TextStyle, ViewStyle } from "react-native"
+
+interface Style {
+    touchableContainer: ViewStyle
+    viewContainer: ViewStyle
+    textStyleName: TextStyle
+    textStyleTask: TextStyle
+}
+
+export default StyleSheet.create<Style>({
+    touchableContainer: {
+        flexDirection: "column",
+        justifyContent: "center"
+    },
+    viewContainer: {
+        paddingLeft: 4
+    },
+    textStyleName: {
```

Comment:
do you need margin between name and task name?

Reply:
Added a small margin, it actually looks quite good, will show you later